src/routes/+page.svelte

```diff
@@ -307,6 +307,27 @@
 	</div>
 </Section>
 
+<!-- About us -->
+<Section id="about-us" class="child:!mx-8">
```

Comment:
I still don't get why you need to change that :/ it looks way too large on desktop screens, and the content is not large enough to absolutely require this. What was your motivation/goal?

Reply:
I made a mistake, I've corrected it, I only wanted this modification for mobile. Because on mobile it's really narrow to read and not pleasant. I can link a screenshot to compare.

Reply:
```suggestion
<Section id="about-us" class="max-sm:child:!mx-8">
```

Reply:
@WarningImHack3r did you approve?

Reply:
Sorry, I didn't get the notification!